### PR TITLE
Adding motivation section to README.

### PR DESCRIPTION
**Pull request by ellisonbg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,30 @@ specification.
...
 For the purpose or prototyping, we are shipping both a Matplotlib renderer and a simple embeddable HTML renderer in Altair.
 
+# Motivation
+
+There are currently many excellent plotting libraries in Python. The main ones are:
+
+* Matplotlib
+* Bokeh
+* Seaborn
+* Lightening
```
Comment (jakevdp) [inline README.md]: sp: Lightning

File: README.md [hunk 2]
```diff
@@ -10,6 +10,30 @@ specification.
...
+* Plotly
+* Pandas built-in plotting
```
Comment (jakevdp) [inline README.md]: Maybe mention holoviews as well, because it tries to do some of the auto-plot-inference we have in mind here

**Comment by jakevdp** [comment]
Just a couple minor comments, but this looks great!

**Comment by ellisonbg** [comment]
Comments addressed and links added. Merging.

**Pull request merged**